examples/showcase/aincraft/components/intersection-spawn.js

```diff
@@ -0,0 +1,36 @@
+/* global AFRAME */
```

Comment:
showcase directory of the examples should illustrate concrete use cases and be distinct. `aincraft` sounds a bit abstract and overlaps a bit with others. I feel these should go into a different directory. Perhaps:

`examples/docs/...`

Reply:
Moved to examples/docs and created a new section "Examples from Documentation" on the examples page.

Will do the same for other PRs, but it will make merging easier if we get this merged to master & then I update the other PRs.